client/src/components/album/AlbumInfo.js

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { endpoint } from '../../helpers/config';
+
+import Button from '../styles/Button';
+
+const AlbumWrapper = styled.div`
+  flex: 1.1;
+  text-align: center;
+`;
+
+const AlbumTitle = styled.h2`
+  font-size: 2.8rem;
+  margin: 0;
+  margin-top: 5px;
+`;
+const ArtistName = styled.p`
+  color: ${props => props.theme.lightWhite} !important;
```

Comment:
`!important` should be avoid if possible. You should be able to achieve the same thing by changing CSS specificity.